config.go

```diff
@@ -109,6 +109,10 @@ type srcfileConfig struct {
 	// to the files in their respective repositories instead of the local copies
 	// inside of Godeps/_workspace.
 	SkipGodeps bool
+
+	// ImportPathRoot is a prefix which is used when converting from repository
+	// URI to Go import path.
+	ImportPathRoot string
```

Comment:
Should we call this `ImportPathHost` and maybe in the doc say something like:  
```
// ImportPathHost, when set, defines the path prefix for this repository's URI in cases 
// when the URI defers from the Go import path.
```
or similar...  

(considering I've understood this correctly)

Although, why would there ever be a difference? Wouldn't that be a problem?